src/suppliers/service/SuppliersService.ts

```diff
@@ -0,0 +1,93 @@
+import { AxiosResponse, AxiosRequestConfig } from 'axios';
+import * as Q from 'q';
+
+import Resource from '../../model/Resource';
+import Supplier from '../model/Supplier';
+import Version from '../../model/Version';
+import RequestClientWrapper from '../../RequestClientWrapper';
+import SuppliersQueryRequestByBusinessId from '../model/SuppliersQueryRequestByBusinessId';
+import RequestCallbackParams from '../../model/RequestCallbackParams';
+import SuppliersResponse from '../model/SuppliersResponse';
+import GenerateApiError from '../../utils/GenerateApiError';
+
+const EXPECTED_GET_STATUS = 200;
+
+export default class SuppliersService {
```

Comment:
Please add a `Suppliers` section to the README.md that outlines `Get supplier` and `Get all suppliers for a business`. It should follow the same format as `Channels` section.